migrations/2021-05-27-154211_account_changes_unique_idx/up.sql

```diff
@@ -1,11 +1,40 @@
+-- This constraint is bad because of 2 reasons:
+-- 1. It does not contain update_reason, so it can remove needed lines
+-- 2. Fortunately, it does not work at all because we have 2 nullable columns, they could not be compared by equality
+--    (all nulls are considered equal)
```

Comment:
I thought all nulls are considered **unique** and that caused the problem, no?

Reply:
Yeap, thanks, that's a typo